### PR TITLE
Added links to npm #742

### PR DESCRIPTION
**Pull request by crew-guy**
Added links to NPM following guidelines given in #763

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,37 +142,60 @@ The `base-icon-btn` class enables you to have a normal, customizable button. Usi
...
-  - [Centered icon rounded button](#icon-center-rounded-button)
-  - [Right icon rounded button](#right-icon-rounded-button)
-- [Block buttons](#block-icon-buttons)
-  - [Left icon block button](#left-icon-block-button)
-  - [Centered icon block button](#icon-center-block-button)
-  - [Right icon block button](#right-icon-block-button)
-
-### Base icon button 
- The `base-icon-btn` class, gives you a default button with no background color and a border. You can choose to style it by adding some of the other available classes, like `btn-orange` to change the background color, and `icon-right` to position the icon in the button. You can make use of it, by copying the sinppet below.
+- [Participate in Hacktoberfest!](#participate-in-hacktoberfest)
```
Comment (shahednasser) [inline README.md:145]: The content menu should not be changed

File: README.md [hunk 2]
```diff
@@ -5,34 +5,48 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
...
 
-**DO NOT submit any pull request that does no fix based on an issue or an actual bug/fix, as it will be labeled invalid and will not count for your Hacktoberfest participation.**  
+**DO NOT submit any pull request that does no fix based on an issue or an actual bug/fix, as it will be labeled invalid and will not count for your Hacktoberfest participation.**
 
 **New rules have been added to Hacktoberfest participation regarding which PRs and which repositories will count to your participation. Since our repository has the topic "hacktoberfest" then we have opted in for participation in Hacktoberfest. any PR that gets merged or approved by a Member will count to your participation. No labels need to be added.**
 
 # Contents
 
+- [Participate in Hacktoberfest!](#participate-in-hacktoberfest)
```
Comment (shahednasser) [inline README.md:25]: Remove any changes in the contents menu

**Review by shahednasser — CHANGES_REQUESTED** [review]
Nice work but I need you to make the changes below and please remove all changes in sbuttons.css as they are unnecessary to this PR

File: README.md (hunk 2 above)
Comment (crew-guy) [reply README.md:25]: Got it. I think my editor itself is "formatting all this on save". Will fix this asap !
How am I screwing up such an easy PR😂🤦🏻‍♂️

**Comment by shahednasser** [comment]
@crew-guy I know you already started over before but I suggest you close this PR, create a new branch, and work on the README from a different editor, as that one is messing up the README. Also, the work between this PR and the other one are getting entangled, so please finish this one first then go back to the other one. I think this will resolve some of the confusion you are having.

**Comment by crew-guy** [comment]
> @crew-guy I know you already started over before but I suggest you close this PR, create a new branch, and work on the README from a different editor, as that one is messing up the README. Also, the work between this PR and the other one are getting entangled, so please finish this one first then go back to the other one. I think this will resolve some of the confusion you are having.

Done !! Check out #781 and #782 !! I really hope I got it all right this time !

**Pull request closed without merging**